updater/lib/tinglesoftware/dependabot/commands/update_all_dependencies_synchronous_command.rb

```diff
@@ -104,12 +104,15 @@ def log_found_open_pull_requests
           end
         end
 
-        def update_all_existing_pull_requests
+        def update_all_existing_pull_requests # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
Just curious where the complexity arises from. I'll investigate later

Reply:
The function already scored 10/10 for that perceived complexity rule. When I added the extra return statement it tipped over to 11/10 and triggered the linting error. 

I _could_ reduce the complexity by splitting the function in to smaller functions, but IMO it is not really adding much value so I decided to ignore it just for that function only.

Reply:
We can just let it stay.